audbackend/core/backend/artifactory.py

```diff
@@ -220,7 +208,9 @@ def _delete(
         self,
     ):
         r"""Delete repository and all its content."""
+        self.open()
```

Comment:
I think it's safer to use a `with` statement here to make sure `close()` is still called when the repository cannot be deleted.

Reply:
I changed to
```python
        with self:
            self._repo.delete()
```